src/app/systelab-components/contextmenu/context-menu-option.ts

```diff
@@ -7,14 +7,15 @@ export type ContextMenuIsIconEnabledFunction = (elementId: string, actionId: str
 export class ContextMenuOption {
 
 	constructor(public actionId: string,
-	            public actionText: string,
-	            public action?: ContextMenuActionFunction,
-	            public isActionEnabled?: ContextMenuIsEnabledFunction,
-	            public isDivider?: boolean,
-	            public iconClass?: string,
-	            public backgroundIconColor?: string,
-	            public iconColor?: string,
-	            public isIconEnabled?: ContextMenuIsIconEnabledFunction) {
+				public actionText: string,
+				public action?: ContextMenuActionFunction,
+				public isActionEnabled?: ContextMenuIsEnabledFunction,
+				public isDivider?: boolean,

```

Comment:
Follow coding conventions. Rename isDivider to hasDivider.

Reply:
The option is a divider itself. I think isDivider is correct